web/src/components/Helmet.tsx

```diff
@@ -26,7 +26,7 @@ const Helmet = ({ pageTitle, metaDescription, languageChangePaths, cityModel }:
 
   const title = `${pageTitle} | ${buildConfig().hostName}`
   const description = metaDescription ?? pageTitle
-  const previewImage = buildConfig().icons.socialMediaPreview
+  const previewImage = `${window.location.origin}${buildConfig().icons.socialMediaPreview}`
```

Comment:
when i parse meta data the social media preview couldn't be resolved with relative link

Reply:
reverted it. error was relying to the nginx header probably. seems like it work in general